### PR TITLE
Add party management service to the services page, targeting #8027.

### PR DESCRIPTION
**Pull request by richardkapolnai-da**
To solve #8027.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/app-dev/services.rst [hunk 1]
```diff
@@ -164,6 +164,18 @@ See :ref:`verbosity` above.
...
 ****************
 
+.. _party-service:
+
+Party management service
+========================
+
+Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.
+
+Allocating parties is necessary to interact with the ledger. See also our :doc:`documentation on Identity Management</concepts/identity-and-package-management>`.
```
Comment (bame-da) [inline docs/source/app-dev/services.rst:174]:
```suggestion
Allocating parties is necessary to interact with the ledger. For more information, refer to the pages on :doc:`Identity Management</concepts/identity-and-package-management>` and :ref:`the API reference documentation <com.daml.ledger.api.v1.admin.PartyManagementService>`.
```

File: docs/source/app-dev/services.rst [hunk 2]
```diff
@@ -164,6 +164,18 @@ See :ref:`verbosity` above.
...
+Party management service offers both retrieving details on an individual party and listing all known parties.
```
Comment (bame-da) [inline docs/source/app-dev/services.rst:175]:
```suggestion
```

File: docs/source/app-dev/services.rst [hunk 3]
```diff
@@ -164,6 +164,18 @@ See :ref:`verbosity` above.
...
+
+For full details, see :ref:`the proto documentation for the service <com.daml.ledger.api.v1.admin.PartyManagementService>`.
```
Comment (bame-da) [inline docs/source/app-dev/services.rst:177]:
```suggestion
```

**Review by bame-da — APPROVED** [review]
thanks. just a few suggestions to remove redundancy

File: docs/source/app-dev/services.rst [hunk 4]
```diff
@@ -164,6 +164,16 @@ See :ref:`verbosity` above.
...
+.. _party-service:
+
+Party management service
+========================
+
+Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.
+
+Allocating parties is necessary to interact with the ledger. For more information, refer to the pages on :doc:`Identity Management</concepts/identity-and-package-management>` and :ref:`the API reference documentation <com.daml.ledger.api.v1.admin.PartyManagementService>`.
+
+
```
Comment (richardkapolnai-da) [inline docs/source/app-dev/services.rst:176]:
```suggestion
```

**Comment by richardkapolnai-da** [comment]
> thanks. just a few suggestions to remove redundancy

thanks, applied them, merging.

**Pull request merged**